### PR TITLE
Fix the learning rate in an audio-classification example

### PR DESCRIPTION
**Pull request by regisss**
# What does this PR do?

The learning rate should be `3e-4` in one of the PyTorch audio classification example according to the link to the corresponding run: https://huggingface.co/anton-l/wav2vec2-base-lang-id

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by anton-l — APPROVED** [review]
Good catch @regisss, thank you!

**Comment by regisss** [comment]
Gently pinging @sgugger for final approval

**Review by sgugger — APPROVED** [review]

**Comment by sgugger** [comment]
Thanks for the fix!
